src/engine/StartupActions.cs

```diff
@@ -15,6 +16,13 @@ private StartupActions()
         // TODO: for devbuilds it would be nice to print the hash here
         GD.Print("This is Thrive version: ", Constants.Version);
 
+        // Add crash logger if debugger not attached
+        if (!Debugger.IsAttached)
+        {
+            GD.Print("Crash logger attached");
```

Comment:
This is not really a crash logger, I'd name this unhandled exception logger, because as I said in my comment, I think none of the reports that players have reported can have been like this. So I'd like to reword this to not confuse players reporting these with the native crashes. So I'd rename all instances of "crash" in this PR to be "Unhandled Exception"